Src/FluentAssertions.Json/JTokenDifferentiator.cs

```diff
@@ -293,47 +269,39 @@ public Difference(DifferenceKind kind, JPath path)
         }
 
         private DifferenceKind Kind { get; }
+
         private JPath Path { get; }
+
         private object Actual { get; }
+
         private object Expected { get; }
 
         public override string ToString()
         {
-            switch (Kind)
+            return Kind switch
```

Comment:
👍🏻 _If_ you need a `switch` statement, this is indeed the best way to represent it.